mantidproject/mantid
# Muon Analysis 2: Output results in a table workspace that aren't numbers

**Description of work.**
Muon Analysis 2 had an issue with plotting a table workspace where a value was a string that was not convertible to a float. This now returns a string.



**To test:**

- Open workbench
- Open Muon Analysis 2
- Load MUSR 62260
- Go to fitting tab
- Fit an Abragam function
- Go to results tab
- Check run_start
- Output table




Fixes #26180. 


NEW FEATURE BUG FIX SO NO RELEASE NOTES

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
